robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+
+Resource       robot/pmm/resources/pmm.robot
+Library        cumulusci.robotframework.PageObjects
+...            robot/pmm/resources/ServiceSchedulePageObject.py
+...            robot/pmm/resources/ServicePageObject.py
+Suite Setup     Run Keywords
+...             Open Test Browser
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+Setup Test Data
+    [Documentation]                 Creates program and service using API
+    ${ns} =                         Get PMM Namespace Prefix
+    Set suite variable              ${ns}
+    ${program} =                    API Create Program
+    Set suite variable              ${program}
+    ${service} =                    API Create Service                  ${Program}[Id]
+    Set suite variable              ${service}
+    ${service_schedule_name} =      Generate New String
+    Set suite variable              ${service_schedule_name}
+    ${today} =                      Get Current Date                    result_format=%-m/%-d/%Y
+    Set suite variable              ${today}
+    ${earlier_date} =               Get Current Date                    result_format=%-m/%-d/%Y            increment=-5 days
+    Set suite variable              ${earlier_date}
+
+*** Test Cases ***
+Validate data on Screen1
+    [Documentation]                        Validates that start date cannot be after end date
+    ...                                    Validates that if Frequency=Daily/Weekly/Monthly, a session end time or session end after is set
```

Comment:
This needs to be removed, I think we will cover this scenario in a separate test

Reply:
@rmonica08 - Do you think I should write it as a separate test? It is already included here.

Reply:
Sorry, got a bit confused. We can leave it in the same test